src/components/FilmsList.vue

```diff
@@ -42,7 +45,6 @@ export default {
   components: { FilmCard, ModalWindow },
   data() {
     return {
```

Comment:
You can use an arrow function for `data`, i.e. `data: () => ({})`

That way you can avoid a `return`.